client/dashboard/me/security-two-step-auth-app/index.tsx

```diff
@@ -21,12 +24,21 @@ export default function SecurityTwoStepAuthApp() {
 			<VStack spacing={ 8 }>
 				{ ! isTwoStepAppEnabled && (
 					<Notice variant="info" title={ __( 'Before you continue' ) }>
-						{
-							// TODO: Add link to support article
+						{ createInterpolateElement(
 							__(
-								'You‘ll need an authenticator app like Google Authenticator or Authy installed on your device to enable two-step authentication.'
-							)
-						}
+								'You‘ll need an authenticator app like Google Authenticator or Authy installed on your device to enable two-step authentication. <learnMoreLink>Learn more</learnMoreLink>'
+							),
+							{
+								learnMoreLink: (
+									<InlineSupportLink
+										supportPostId={ 58847 }
+										supportLink={ localizeUrl(
+											'https://wordpress.com/support/security/two-step-authentication/#use-an-app'
+										) }
+									/>
+								),
+							}
+						) }
```

Comment:
Updated the link

Reply:
<img width="686" height="252" alt="Screenshot 2025-09-25 at 12 45 46 PM" src="https://github.com/user-attachments/assets/22f49c28-4e2d-49dd-8b91-ac261f18a155" />